impeller/typographer/rectangle_packer.h

```diff
@@ -50,6 +50,19 @@ class RectanglePacker {
   ///
   virtual float percentFull() const = 0;
 
+  //----------------------------------------------------------------------------
+  /// @brief     Create a new rectangle packer with a larger scaled height
+  ///            scaled and initialize its contents to the current packer.
+  ///
+  /// @param[in] scale  The scaling factor to be applied to the new height.
+  ///
+  /// @return    A new rectangle packer.
+  ///
+  ///            This method is used for growing the glyph atlas while keeping
+  ///            existing glyphs in place. The width of the rectangle packer
+  ///            cannot be increased.
+  virtual std::unique_ptr<RectanglePacker> Clone(int scale) = 0;
```

Comment:
Nit: Consider uint32_t for the scale as a negative scale makes no sense.

Reply:
Done